packages/react/examples/create-react-app/README.md

```diff
@@ -5,7 +5,13 @@
 
 ## Usage
 
-To run this example, all you need to do is `cd` into the directory, run
+First run `yarn build` in the root of the `carbon` repository.
+
+```sh
+yarn install && yarn build
+```
+
+Now to run this example, all you need to do is `cd` into the directory, run
```

Comment:
I think we should add this "`cd` into the directory" instruction to the other readmes as well before instructing to `yarn install`

Reply:
Cool! I like the idea.